models/classes/LaunchData/Validator/LaunchDataValidator.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 (original work) Open Assessment Technologies SA
+ */
+
+namespace oat\taoLti\models\classes\LaunchData\Validator;
+
+
+use oat\taoLti\models\classes\LtiException;
+use oat\taoLti\models\classes\LtiInvalidLaunchDataException;
+use oat\taoLti\models\classes\LtiVariableMissingException;
+use oat\taoLti\models\classes\LtiLaunchData;
+
+/**
+ * Class LaunchDataValidator
+ * @package oat\taoLti\models\classes\LaunchData\validator
+ */
+class LaunchDataValidator
```

Comment:
We should maybe put version name in the class name because we will have a different class for a different version or maybe put the pattern in configuration so it can fit any version

Reply:
done